python/itt_NERSC/__init__.py

```diff
@@ -0,0 +1,8 @@
+"""
+Python binding to NERSC implementation of Intel Instrumentation and Tracing Technology (ITT) API.
+"""
```

Comment:
Just a thought from an interested observer: I think the history of this NERSC implementation would be _very_ helpful to understand. I'm asking questions like, "what is NERSC?" and "why would they only want to use a subset of ittapi?" and "is there something wrong with ittapi that requires a compatibility shim?" I can figure out some of the answers to these questions, or at least guess at them, but if you explained that up front, clearly, in the documentation (and maybe in the README) it would eliminate some of that confusion.

Reply:
I do agree that it is not clearly documented at the moment. We call that NERSC (National Energy Research Scientific Computing Center) implementation because the very first ITT bindings for Python were developed at https://github.com/NERSC/itt-python, that is now moved to https://github.com/oleksandr-pavlyk/itt-python and there were already a number of users of that API, and even VTune's own cookbook recipes were referring to it. Since we have introduced our own (arguably) superior implementation of bindings, we wanted to minimize the effort for transitioning for the existing users of itt-python if they wish to do so by implementing this compatibility layer that would just be a drop-in replacement for itt-python module that would use ittapi bindings internally.

@mk-srivastava could you please also update the README.md with some info about the compatibility layer? At least to point out that it exists, what is it for and how to use it.

Reply:
Will do the update in README.md about the compatibility layer.